File: Fundamentals_of_Software_Engineering/front/lk/common/src/commonMain/kotlin/org/suai/common/model/specialityModel.kt
package org.suai.common.model

/**
 * @param Id id специальности
 * @param Name название специальности
 * @param Code код специальности
 * @param educationType тип программы обучения
**/
data class specialityModel(
    val Id:   Int                       = 0,
    val Code: String                    = "",
    val Name: String                    = "",
    val educationType:  EDUCATION_TYPE  = EDUCATION_TYPE.NONE,
)

File: Fundamentals_of_Software_Engineering/back/lkAPI/lkAPI/Models/SpecialityModel.cs
using lkAPI.Models.Enums;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace lkAPI.Models
{
    public class SpecialityModel : EntityBase
    {
        public virtual string code { get; set; }
        public virtual string name { get; set; }
        public virtual EducationType educationType { get; set; }
    }
}
